tests/unit/source_code/test_notebook.py

```diff
@@ -0,0 +1,297 @@
+from collections.abc import Callable
+
+import pytest
+from databricks.sdk.service.workspace import Language
+
+from databricks.labs.ucx.source_code.notebook import Notebook, NotebookDependencyGraph
+from tests.unit import _load_sources
+
+PYTHON_NOTEBOOK_SAMPLE = (
+    "00_var_context.py.sample",
+    Language.PYTHON,
+    ['md', 'md', 'md', 'python', 'python', 'python', 'md', 'python', 'md'],
```

Comment:
can you remove the irrelevant test cells?

Reply:
mmm... it was actually super useful to use real notebooks to discover edge cases, and testing against an untouched sample creates a lot of confidence

Reply:
it'd be much less intrusive if `make fmt` didn't force 1 item per line. Is there a way to configure this ?

Reply:
Fixed by disabling fmt for these lines